### PR TITLE
May This Month in Solid Newsletter

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**

### CODE REVIEW

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Christian Buggedei, Walter Almeida, Glen Simister, Manoharan Ramachandran, and Michiel de Jong for presenting at Solid World May last Thursday.
+
+In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you.Of note, Darcy recently won a Mozilla grant. You can find the Darcy source code here. Christian Buggedei is a founder and technology expert.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:18]:
```suggestion
In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you. Of note, Darcy recently won a Mozilla grant. You can find the Darcy source code here. Christian Buggedei is a founder and technology expert.
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages dbPedia.org as a data source for countries and places. You can find the Go location app source code here.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:20]: *DBpedia (or just dbpedia)

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+In the third talk, Manoharan Ramachandran presented the Solid app built by Open University's blockchain team, looking into how to support immunity passports as mentioned by the UK government daily briefings. The Solid app facilitates the instant verification of tamper-proof test results whilst preserving privacy.
+
+In the fourth talk, Glen Simister presented the DVO Profile Manager, a public profile (or resume), which anyone can create for free, and update. DVO Profile Manager works in conjunction with the DVO Toolbar, a browser extension which allows people to like, dislike, and comment on any URL. You can find the DVO Profile Manager source code here.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:24]: missing link

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+In the fifth talk, Michiel de Jong presented his latest experiment, SNAP on Solid, explaining how Solid inboxes can be used as the messaging layer for Network Ledger Technology. Check out the SNAP source code here.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:26]: missing link

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+#### Solid Implementations
+
+* [Solid Health](https://github.com/jasonpaulos/solid-health)
+* [DVO](https://dvo1.github.io/profile/index.html)
+* [Snap](https://michiels-nlt-kit.herokuapp.com)
+* [Darcy](https://ibex.darcy.is/feed)
+
+#### External Grants
+
+There are an EU initiative [DAPSI](https://dapsi.ngi.eu/) and [NGI Pointer](https://www.ngi.eu/ngi-projects/ngi-pointer/) that could be suitable for Solid app projects.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:50]:
```suggestion
There are EU initiatives [DAPSI](https://dapsi.ngi.eu/) and [NGI Pointer](https://www.ngi.eu/ngi-projects/ngi-pointer/) that could be suitable for Solid app projects.
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+* [Minutes](https://www.w3.org/community/solid/wiki/Meetings#2020-04-08T13:30:00Z) from April 8th
+* [Minutes](https://www.w3.org/community/solid/wiki/Meetings#2020-04-24_1600CET) from April 24th
+
+Other notable activity related to the Solid specification include:
+
+* A draft candidate proposal for WebID-OIDC is underway in the [Authentication Panel](https://github.com/solid/authentication-panel) based on rough consensus agreement around key elements of the protocol needed to support the development of reference implementations.
+* Use cases for server managed auxiliary resources related to resource provenance, auditing, and access history are being undertaken in the [Data Interoperability Panel](https://github.com/solid/data-interoperability-panel).
+* Work towards a normative specification for [web access control](https://github.com/solid/web-access-control-spec) will begin soon, organized through the [Authorization Panel](https://github.com/solid/authorization-and-access-control-panel).
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:73]:
```suggestion
* Work towards a normative specification for [Web Access Control](https://github.com/solid/web-access-control-spec) will begin soon, organized through the [Authorization Panel](https://github.com/solid/authorization-and-access-control-panel).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 2 above)
Comment (Vinnl) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:20]:
```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the Go location app source code here.
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 3 above)
Comment (Vinnl) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:24]:
```suggestion
In the fourth talk, Glen Simister presented the DVO Profile Manager, a public profile (or resume), which anyone can create for free, and update. DVO Profile Manager works in conjunction with the DVO Toolbar, a browser extension which allows people to like, dislike, and comment on any URL. You can find [the DVO Profile Manager source code on GitHub](https://github.com/dvo/profile).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 4 above)
Comment (Vinnl) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:26]:
```suggestion
In the fifth talk, Michiel de Jong presented his latest experiment, SNAP on Solid, explaining how Solid inboxes can be used as the messaging layer for Network Ledger Technology. Check out the [SNAP source code on GitHub](https://github.com/ledgerloops/snap-solid/).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+In the third talk, Manoharan Ramachandran presented the Solid app built by Open University's blockchain team, looking into how to support immunity passports as mentioned by the UK government daily briefings. The Solid app facilitates the instant verification of tamper-proof test results whilst preserving privacy.
+
+In the fourth talk, Glen Simister presented the DVO Profile Manager, a public profile (or resume), which anyone can create for free, and update. DVO Profile Manager works in conjunction with the DVO Toolbar, a browser extension which allows people to like, dislike, and comment on any URL. You can find the DVO Profile Manager source code here.
+
+In the fifth talk, Michiel de Jong presented his latest experiment, SNAP on Solid, explaining how Solid inboxes can be used as the messaging layer for Network Ledger Technology. Check out the SNAP source code here.
+
+If you missed Solid World you can [watch the recording [here](https://vimeo.com/415976385).
```
Comment (Vinnl) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:28]: There's a duplicate `[` here:

```suggestion
If you missed Solid World you can [watch the recording on Vimeo](https://vimeo.com/415976385).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+The [next Solid World](https://www.eventbrite.com/e/solid-world-june-tickets-104631158612) will happen soon on the 4th June  2020. If you have any content requests or would like to present at the next Solid world email m.laszlo@inrupt.com.
```
Comment (Vinnl) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:30]: Remove duplicate space, add link:

```suggestion
The [next Solid World](https://www.eventbrite.com/e/solid-world-june-tickets-104631158612) will happen soon on the 4th June 2020. If you have any content requests or would like to present at the next Solid world email [m.laszlo@inrupt.com](mailto:m.laszlo@inrupt.com).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 2 above)
Comment (RubenVerborgh) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:20]: missing link still

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 1 above)
Comment (Vinnl) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:18]:
```suggestion
In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you. Of note, Darcy recently won a Mozilla grant. You can find [the Darcy source code on GitHub](https://github.com/Darcy-Social/ibex). Christian Buggedei is a founder and technology expert.
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md (hunk 2 above)
Comment (Vinnl) [reply _posts/newsletter/2020-05-07-this-month-in-solid.md:20]: Right, updated suggestion:

```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).
```

File: _posts/newsletter/2020-05-07-this-month-in-solid.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Christian Buggedei, Walter Almeida, Glen Simister, Manoharan Ramachandran, and Michiel de Jong for presenting at Solid World May last Thursday.
+
+In the first talk, Christian Buggedei presented Darcy, a social network that knows nothing about you. Of note, Darcy recently won a Mozilla grant. You can find the Darcy source code here. Christian Buggedei is a founder and technology expert.
+
+In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and Sparql endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-05-07-this-month-in-solid.md:20]:
```suggestion
In the second talk, Walter Almeida presented GO for modeling and generating Solid apps quickly with minimum code. He demonstrated the GO connector between Pod and SPARQL endpoints using a location tracking Solid app that leverages DBpedia as a data source for countries and places. You can find the [Go location app source code on GitHub](https://github.com/walteralmeida/GOForSolidVisitedPlacesApplication/).
```

**Pull request merged**